doc/tutorial/quick-start.md

```diff
@@ -0,0 +1,34 @@
+# Quick start
+
+## What you'll do
+
+- Install the CLI
+- Initialize the builder
+- Run the image build
+
+## Requirements
+
+- [Pipx installed](https://pipx.pypa.io/stable/installation/)
+- Apt packages gcc, pipx, python3-dev
+  - `sudo apt-get install -y python3-dev gcc pipx`
+- Working [OpenStack environment](https://microstack.run/docs/single-node)
+- A clouds.yaml configuration with the OpenStack environment
+
+## Steps
+
+### Install the CLI
+
+- Install the CLI
+  - `pipx install git+https://github.com/canonical/github-runner-image-builder@stable`
+
+### Initialize the builder
+
+- `github-runner-image-builder init`
+
+### Run the image build
+
+- `github-runner-image-builder run <cloud-name> <image-name>`
```

Comment:
How does the user get or choose the `<cloud-name>` and `<image-name>`?

Reply:
Added a bit more context, openstack cloud-name bit requires a bit of cloud domain knowledge (link to https://microstack.run/docs/single-node sets the cloud up for running it).
What would your approach be?